pkg/operator/workloads/api_workload.go

```diff
@@ -291,6 +291,17 @@ func tfAPISpec(
 						Env:          k8s.AWSCredentials(),
 						VolumeMounts: k8s.DefaultVolumeMounts(),
 					},
+					{
+						Name:            downloaderInitContainerName + "-project",
+						Image:           config.Cortex.DownloaderImage,
+						ImagePullPolicy: "Always",
+						Args: []string{
+							"--download_from=" + config.AWS.S3Path(ctx.ProjectKey),
+							"--download_to=" + path.Join(consts.EmptyDirMountPath, "project"),
```

Comment:
how do you feel about combining these into a single init container which downloads both? We probably don't need to pay the container overhead twice here